catroid/src/androidTest/java/org/catrobat/catroid/test/io/asynctask/CopyProjectToExternalStorageAsyncTaskTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.test.io.asynctask;
+
+import android.Manifest;
+import android.app.NotificationManager;
+import android.content.Context;
+import android.os.AsyncTask;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.rule.GrantPermissionRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.util.Pair;
+import android.util.SparseArray;
+
+import org.catrobat.catroid.ProjectManager;
+import org.catrobat.catroid.content.Project;
+import org.catrobat.catroid.content.Scene;
+import org.catrobat.catroid.io.StorageOperations;
+import org.catrobat.catroid.io.asynctasks.CopyProjectToExternalStorageAsyncTask;
+import org.catrobat.catroid.test.utils.Reflection;
+import org.catrobat.catroid.utils.NotificationData;
+import org.catrobat.catroid.utils.StatusBarNotificationManager;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.File;
+
+import static org.catrobat.catroid.common.Constants.CATROBAT_EXTENSION;
+import static org.catrobat.catroid.common.FlavoredConstants.EXTERNAL_STORAGE_ROOT_EXPORT_DIRECTORY;
+@RunWith(AndroidJUnit4.class)
+public class CopyProjectToExternalStorageAsyncTaskTest {
```

Comment:
please use export over copy in all the method and classnames. (since weve already have so many "copySomething" methods, and this one here does not just copy)